app/js/controllers/main.js

```diff
@@ -92,4 +96,36 @@ export default class MainController extends BaseController {
         console.error('Error while subscribing to notifications:', err);
       });
   }
+
+  /**
+   * Will init the `voiced` property, which controls whether a notification is
+   * spoken.
+   *
+   * This works by using a specific hash while loading the application.
+   * Recognized syntaxes are: voiced=1/true/0/false
+   *
+   * Note that we don't change the property right after changing the hash (no
+   * hashchange event handler), so you need to reload the application with the
+   * hash specified.
+   */
+  [p.initVoiced]() {
+    const forcedMatch = location.hash.match(/\bvoiced=(.+?)(&|$)/);
+
+    if (forcedMatch) {
+      const matchedValue = forcedMatch[1];
+      const willSpeak = matchedValue === '1' || matchedValue === 'true';
+
+      console.log('Will force the `voiced` property to ', willSpeak);
+      if (willSpeak) {
+        localStorage['voiced'] = 'true'; // localStorage values are strings
```

Comment:
Can you reuse the `lib/server/settings.js` class for that? The best approach is to expose a getter/setter in the `Server` class that interacts directly with its private instance of `Settings`:
```js
this[p.server].voiced = willSpeak;
```

You'll get the benefits that you don't have to cast to strings, you set a default value (so it won't always have to be stored in the local storage) and this preference will be erased when calling `this[p.server].clear()`.